File: Backend/user-service/spring_boot_backend_template/spring_boot_backend_template/src/main/java/com/backend/dto/BusScheduleDTO.java
package com.backend.dto;

import lombok.Data;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDate;
import java.time.LocalTime;
import java.util.List;

@Getter
@Setter
public class BusScheduleDTO {

    private Long busId;

    private String fromCity;
    private String toCity;
    private LocalDate journeyDate;
    private LocalTime departureTime;
    private LocalTime arrivalTime;
    private double ticketPrice;

    private List<BoardingPointRequestDTO> boardingPoints;
    private List<DroppingPointRequestDTO> droppingPoints;
}

File: Backend/admin-service/admin/admin-service/DTO/BusScheduleDTO.cs
namespace admin_service.DTO
{
    using System;

    namespace admin_service.DTO
    {
        public class BusScheduleDto
        {
            public long Id { get; set; }
            public long BusId { get; set; }
            public string FromCity { get; set; }
            public string ToCity { get; set; }
            public DateTime JourneyDate { get; set; }
            public TimeSpan DepartureTime { get; set; }
            public TimeSpan ArrivalTime { get; set; }
            public decimal TicketPrice { get; set; }
            public string Status { get; set; }
        }
    }

}
